src/main/java/life/qbic/portal/portlet/ImageInfo.java

```diff
@@ -0,0 +1,54 @@
+package life.qbic.portal.portlet;
+
+/**
+ * <short description>
+ * <p>
+ * <detailed description>
+ *
```

Comment:
What is the intention of the class? A short description could help me to better review your code :)

Reply:
9fd8aa1